internal/planner/filter/copy_field.go

```diff
@@ -80,6 +80,14 @@ func traverseFilterByProperty(
 				} else if shouldDelete {
 					delete(result, opKey)
 				}
+
+			case map[connor.FilterKey]any:
```

Comment:
suggestion: would be great to have this one tested in `copy_field_test.go`

Reply:
I've added a test. There was potentially a bug with `_not` filters before I modified it.